agent/06_agent_create_cluster.sh

```diff
@@ -528,14 +604,14 @@ if [[ ! -z $AGENT_OPERATORS ]]; then
     put_operator_file
 fi
 
+
 wait_for_cluster_ready
```

Comment:
Seems that a changed is needed [here](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/06_agent_create_cluster.sh#L219) since the `openshift-install` path isn't created. I'm getting
```
Assisted install UI is up: http://192.168.111.20:3001
./agent/06_agent_create_cluster.sh: line 273: /home/stack/dev-scripts-bfournie-ove/ocp/ostest/openshift-install: No such file or directory
```

Reply:
The code execution should skip after the UI is up and the error you saw was due to this [typo](https://github.com/openshift-metal3/dev-scripts/pull/1745/files#diff-51babdd488dfb3c638b47aaeb87403b34eb32bed9422adf3632ac9372460ebc7R572) This will be fixed in upcoming rebase